src/cljs/athens/devcards/app_toolbar.cljs

```diff
@@ -193,8 +202,13 @@
             :content [:div (use-style modal-contents-style)
                       ;; TODO: Write intro copy
                       [:p "Some helpful framing about what Athens does and what users should expect. Athens is not Roam."]
+                      [:p "To export a Roam database to the " [:code "json triplet"] " format that Athens understands, refer to the official Athens documentation."]
                       [features-table]
-                      ;; TODO: Create browser file dialog and actually import stuff
-                      [:div [button {:primary true} "Add Files"]]]
+                      [:input {:type "file"
```

Comment:
Adds a hidden db import field so that we can upload a file using a single button only.

Reply:
https://stackoverflow.com/a/38921114 but this is '16 so I don't know if things changed yet